compat/src/index.js

```diff
@@ -124,6 +124,14 @@ function createElement(...args) {
 			delete props.defaultValue;
 		}
 
+		if (Array.isArray(props.value) && props.multiple && type==='select') {
+			props.children.forEach((child) => {
+				if (props.value.includes(child.props.value)) {
```

Comment:
Let's use `indexOf` instead. We officially support browsers all the way down to IE11 and unofficially even older. They have no support for `Array.prototyp.includes`